.github/workflows/release-container-image.yml

```diff
@@ -0,0 +1,237 @@
+name: Release multi-arch container image
```

Comment:
It would be great to be able to reference this from other workflows. You can find more about how to do that [in the following documentation](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow)

In my opinion, the inputs should be the ones you specify in the `env` section. The need for me is to be able to reuse this workflow building the image on multiple platforms, adding a signature to it in an automated way (probably in the tests themselves).

Reply:
The goal is to be able to run this with a slightly different target, for the upcoming tests on ICUs, where the image will need to be published somewhere so we can test the overall image release and install cycle works, so I would say it's a reusable workflow with `workflow_call`

Reply:
Done in 2e2b6cf30841aee7b887816adab98c6164f26574.

Reply:
Eww yeah!